pybamm/models/submodels/interface/kinetics/total_main_kinetics.py

```diff
@@ -34,39 +32,18 @@ def get_coupled_variables(self, variables):
         Domain = self.domain
         domain = Domain.lower()
 
-        if Domain == "Negative" and self.half_cell is True:
-            return variables
-
         phases = self.options.phases[domain]
-        if len(phases) > 1:
-            a_j = sum(
-                variables[
-                    f"{Domain} electrode {phase} volumetric interfacial current density"
-                ]
-                for phase in phases
-            )
-            a_j_dim = sum(
-                variables[
-                    f"{Domain} electrode {phase} volumetric "
-                    "interfacial current density [A.m-3]"
-                ]
-                for phase in phases
-            )
-            a_j_av = sum(
-                variables[
-                    f"X-averaged {domain} electrode {phase} volumetric "
-                    "interfacial current density"
-                ]
-                for phase in phases
-            )
-            variables.update(
-                {
-                    f"{Domain} electrode volumetric interfacial current density": a_j,
-                    f"{Domain} electrode volumetric "
-                    "interfacial current density [A.m-2]": a_j_dim,
-                    f"X-averaged {domain} electrode volumetric "
-                    "interfacial current density": a_j_av,
-                }
+        # For each of the variables, the variable name without the phase name
+        # is constructed by summing all of the variable names with the phases
+        for variable_template in [
+            f"{Domain} electrode {{}}volumetric interfacial current density",
+            f"{Domain} electrode {{}}volumetric " "interfacial current density [A.m-3]",
```

Comment:
why the quotes after volumetric?

Reply:
typo :)

Reply:
it happens when strings need to be split over two lines and then later fit on one line